generator/source/faq/installation.rst

```diff
@@ -279,7 +270,7 @@ It is just a warning and you can ignore it. Verify the Sawtooth repository was a
 
 I get this error starting Sawtooth on Docker: ``No response from ... beginning heartbeat pings``
 ------------------------------------------------------------------------------------------------
-This means there is a problem with the genesis node and peer nodes connecting.
+It means that there has been no message exchange between the validators for 10 seconds. There is not necessarily a problem, but it could mean the the genesis node and peer nodes are not connecting.
```

Comment:
Genesis nodes and peer nodes are not connecting?
It could be on any of the validators. Seeing this trace is a problem in case of Raft/PBFT but not with other consensus engines.

Reply:
LGTM. I feel it's better to get the wording reviewed by one more approver.